### PR TITLE
Remove tedious, incomplete and unmaintained list of events per interface

### PR DESCRIPTION
**Pull request by teoli2003**
This PR removes the list of events per interface because:
- it duplicates information that is in each interface now (it wasn't initially the case)
- it is incomplete
- it is impossible to maintain (far too complex)

I also fixed a broken link (not listed as a flaw) in the table for Performance API: We now have an overview page that groups all the related specs. It is shorter, more convenient, and will always be up-to-date.

### CODE REVIEW

**Comment by estelle** [comment]
Should we be saying "Performance APIs", with an "s", here and on https://developer.mozilla.org/en-US/docs/Web/API/Performance_API?

**Comment by teoli2003** [comment]
cc/ @elchi3 who created the Performance API page and is likely to have a better insight than me.

**Comment by Elchi3** [comment]
> cc/ @Elchi3 who created the Performance API page and is likely to have a better insight than me.

I have no strong feelings. The term "API" is overloaded and it is rarely a single "interface" that is being referred to anyways.

File: files/en-us/web/events/index.md [hunk 1]
```diff
@@ -556,19 +556,9 @@ This topic provides an index to the main _sorts_ of events you might be interest
...
-          <a href="/en-US/docs/Web/API/Performance_Timeline"
-            >Performance Timeline API</a
-          >,
-          <a href="/en-US/docs/Web/API/Navigation_timing_API"
-            >Navigation Timing API</a
-          >, <a href="/en-US/docs/Web/API/User_Timing_API">User Timing API</a>,
-          and
-          <a href="/en-US/docs/Web/API/Resource_Timing_API"
-            >Resource Timing API</a
+            >Performance API</a
```
Comment (teoli2003) [inline files/en-us/web/events/index.md:561]:
```suggestion
            >Performance APIs</a
```

**Comment by teoli2003** [comment]
Ok, I made a call and added the `s`. Not a biggie though.

**Review by estelle — APPROVED** [review]

**Comment by teoli2003** [comment]
Thank you!

**Pull request merged**